druid/src/main/java/org/apache/calcite/adapter/druid/DruidQuery.java

```diff
@@ -913,6 +917,19 @@ String translate(RexNode e, boolean set) {
         case LESS_THAN_OR_EQUAL:
           return new JsonBound("bound", tr(e, posRef), null, false, tr(e, posConstant), false,
               call.getOperands().get(posRef).getType().getFamily() == SqlTypeFamily.NUMERIC);
+        case IN:
+          ImmutableList.Builder listBuilder = ImmutableList.builder();
+          for (RexNode rexNode: call.getOperands()) {
+            if (rexNode.getKind().equals(SqlKind.LITERAL)) {
+              listBuilder.add(((RexLiteral) rexNode).getValue2().toString());
+            }
+          }
+          return new JsonInFilter("in", tr(e, posRef), listBuilder.build());
+        case BETWEEN:
+          return new JsonBound("bound", tr(e, posRef), tr(e, posConstant), false,
```

Comment:
I think this will fail, since BETWEEN first parameter is whether it is negated or not, then the reference, and then the bounds (thus, 4 arguments in total). You can check ```org.apache.calcite.sql.fun.SqlBetweenOperator```.